src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/NaturalForeignKeyValidation.feature.disabled

```diff
@@ -0,0 +1,189 @@
+Feature: Validation of Natural and Foreign Keys
```

Comment:
The last scenario is a duplicate and can be deleted. Or if I'm wrong and its not a duplicate, it can move to `InvalidReferenceValidation.feature`.

Then you can change the name and the Feature for this file. All of the remaining scenarios are about "key unification", so let's make that very clear.

Call the file `KeyUnification.feature` and change to `Feature: Validation of Natural Key Unification`

Reply:
File name changed, as well as the feature